airflow-core/src/airflow/cli/commands/dag_command.py

```diff
@@ -648,6 +649,149 @@ def dag_test(args, dag: DAG | None = None, session: Session = NEW_SESSION) -> No
+            if seconds >= 86400:  # 24 hours
+                days = int(seconds // 86400)
+                return f"Every {days} day{'s' if days > 1 else ''}"
+            elif seconds >= 3600:  # 1 hour
+                hours = int(seconds // 3600)
+                return f"Every {hours} hour{'s' if hours > 1 else ''}"
+            elif seconds >= 60:  # 1 minute
+                minutes = int(seconds // 60)
+                return f"Every {minutes} minute{'s' if minutes > 1 else ''}"
+            else:
+                return f"Every {int(seconds)} second{'s' if seconds > 1 else ''}"
+        
+        # Handle timetable objects
+        if hasattr(dag, 'timetable') and dag.timetable:
+            timetable = dag.timetable
+            # Try to get a description from the timetable
+            if hasattr(timetable, 'description'):
+                return timetable.description
+            elif hasattr(timetable, 'summary'):
+                return timetable.summary
+            else:
+                return str(timetable.__class__.__name__)
+        
+        # Handle dataset schedules
+        if hasattr(schedule, '__iter__') and not isinstance(schedule, str):
+            # Likely a dataset schedule (list/set of datasets)
+            try:
+                datasets = list(schedule)
+                if datasets:
+                    dataset_names = [str(ds) for ds in datasets[:3]]  # Show first 3
+                    if len(datasets) > 3:
+                        return f"Dataset-triggered: {', '.join(dataset_names)}, ..."
+                    else:
+                        return f"Dataset-triggered: {', '.join(dataset_names)}"
+            except (TypeError, AttributeError):
+                pass
+        
+        # Fallback to string representation
+        return str(schedule)
+        
+    except Exception:
+        # If anything fails, return None to skip schedule info
+        return None
+
+
+@cli_utils.action_cli
+@suppress_logs_and_warning
+@providers_configuration_loaded
+def dag_docs(args) -> None:
+    """Display DAG documentation (__doc_md__) at the command line."""
+    if args.dag_id:
+        # Get specific DAG
+        dag = get_dag(args.subdir, args.dag_id)
+        dags = [dag]
+    else:
+        # Get all DAGs
+        dagbag = DagBag(process_subdir(args.subdir))
+        if dagbag.import_errors:
+            from rich import print as rich_print
+
+            rich_print(
+                "[red][bold]Error:[/bold] Failed to load all files. "
+                "For details, run `airflow dags list-import-errors`",
+                file=sys.stderr,
+            )
+        dags = sorted(dagbag.dags.values(), key=operator.attrgetter("dag_id"))
+
+    # Generate markdown output
+    markdown_output = []
+    
+    if len(dags) > 1:
+        markdown_output.append("# DAG Documentation\n")
+    
+    for dag in dags:
+        doc_md = getattr(dag, '__doc_md__', None) or getattr(dag, 'doc_md', None)
```

Comment:
```suggestion
        doc_md = getattr(dag, '__doc_md__', getattr(dag, 'doc_md', None))
```

no strong opinion on this one.